00-introduction.md

```diff
@@ -98,179 +98,180 @@ See [BOLT #11: Invoice Protocol for Lightning Payments](11-payment-encoding.md)
+   * A conditional payment between two *[peers](#peers)*: the recipient can spend
     the payment by presenting its signature and a *payment preimage*,
     otherwise the payer can cancel the contract by spending it after
     a given time. These are implemented as outputs from the
-    *commitment transaction*.
-   * _See container: commitment transaction_
-   * _See parts: Payment hash, Payment preimage_
+    *[commitment transaction](#commitment-transaction)*.
+   * _See container: [commitment transaction](#commitment-transaction)_
+   * _See parts: [Payment hash](#Payment-hash), [Payment preimage](#Payment-preimage)_
 
-* *Invoice*: A request for funds on the Lightning Network, possibly
+* #### *Invoice*: A request for funds on the Lightning Network, possibly
     including payment type, payment amount, expiry, and other
     information. This is how payments are made on the Lightning
     Network, rather than using Bitcoin-style addresses.
 
-* *It's ok to be odd*:
+* #### *It's ok to be odd*:
    * A rule applied to some numeric fields that indicates either optional or
      compulsory support for features. Even numbers indicate that both endpoints
      MUST support the feature in question, while odd numbers indicate
      that the feature MAY be disregarded by the other endpoint.
 
-* *MSAT*:
+* #### *MSAT*:
    * A millisatoshi, often used as a field name.
 
-* *Mutual close*:
-   * A cooperative close of a *channel*, accomplished by broadcasting an unconditional
-    spend of the *funding transaction* with an output to each *peer*
+* #### *Mutual close*:
+   * A cooperative close of a *[channel](#channel)*, accomplished by broadcasting an unconditional
+    spend of the *[funding transaction](#funding-transaction)* with an output to each *peer*
     (unless one output is too small, and thus is not included).
-   * _See related: revoked transaction close, unilateral close_
+   * _See related: [revoked transaction close](#revoked-transaction-close), [unilateral close](#unilateral-close)_
 
-* *Node*:
+* #### *Node*:
    * A computer or other device that is part of the Lightning network.
-   * _See related: peers_
-   * _See types: final node, hop, origin node, processing node, receiving node, sending node_
+   * _See related: [peers](#peers)_
+   * _See types: [final node](#final-node), [hop](#hop), [origin node](#origin-node), [processing node](#processing-node), [receiving node](#receiving-node), [sending node](#sending-node)_
 
-* *Origin node*:
-   * The _node_ that originates a packet that will route a payment through some number of _hops_ to a _final node_. It is also the first _sending peer_ in a chain.
-   * _See category: node_
-   * _See related: final node, processing node_
+* #### *Origin node*:
+   * The *[node](#node)* that originates a packet that will route a payment through some number of [hops](#hop) to a *[final node](#final-node)*. It is also the first _sending peer_ in a chain.
```

Comment:
nit: you can also add a link for `sending peer`

Reply:
Done ;)